pkg/types/baremetal/validation/libvirt.go

```diff
@@ -0,0 +1,77 @@
+// +build baremetal
+
+package validation
+
+import (
+	"fmt"
+	"github.com/libvirt/libvirt-go"
+	"github.com/openshift/installer/pkg/types/baremetal"
+	"github.com/pkg/errors"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+	"strings"
+)
+
+func init() {
+	baremetalValidators = append(baremetalValidators, validateInterfaces)
+}
+
+// validateInterfaces ensures that any interfaces required by the platform exist on the libvirt host.
+func validateInterfaces(p *baremetal.Platform, fldPath *field.Path) field.ErrorList {
+	errorList := field.ErrorList{}
+
+	findInterface, err := interfaceValidator(p.LibvirtURI)
+	if err != nil {
+		errorList = append(errorList, field.InternalError(fldPath.Child("libvirtURI"), err))
+		return errorList
```

Comment:
Error looks like:

```
FATAL failed to fetch Terraform Variables: failed to load asset "Install Config": invalid "install-config.yaml" file: platform.baremetal.libvirtURI: Internal error: could not connect to libvirt: virError(Code=38, Domain=7, Message='Failed to connect socket to '/var/run/libvirt/libvirt-sock': Connection refused')
```

Reply:
Instead of providing these here, maybe a unit-test case is more appropriate? and if that's not easy include these errors in the commit message.